src/applications/vaos/appointment-list/components/AppointmentsPage/index.jsx

```diff
@@ -1,35 +1,29 @@
-import React, { useEffect, useState } from 'react';
-import { useSelector, useDispatch } from 'react-redux';
-import { Switch, Route, useLocation, useHistory } from 'react-router-dom';
-import classNames from 'classnames';
 import DowntimeNotification, {
   externalServices,
 } from '@department-of-veterans-affairs/platform-monitoring/DowntimeNotification';
+import classNames from 'classnames';
```

Comment:
Ooo this is nice, what did you use to sort imports?

Reply:
:smiley: You noticed! Option + Shift + O in VSCode. You can also get to it from the source context menu->Source Action->Organize Imports.